pages/people_page.py

```diff
@@ -54,3 +56,30 @@ def we_filter_incorrect(self):
             print("\tData filtered correctly. Incorrect result was not displayed")
             self.context.wait.until(EC.element_to_be_clickable((By.XPATH, '//span[@title="Clear search"]')))
             self.context.driver.find_element_by_xpath('//span[@title="Clear search"]').click()
+
+    def create_list_name(self):
+        list_name = []
+        for i in self.context.driver.find_elements_by_css_selector('a.name'):
+            list_name.append(i.text)
+        return list_name
+
+    def set_list_name(self):
+        self.list_name = self.create_list_name()
+
+    def create_alphabet(self):
+        alphabet = []
+        for i in self.context.driver.find_elements_by_css_selector('li>a.pagination-item'):
+            alphabet.append(i.text)
+        return alphabet
+
+    def choose_letter(self):
+        self.random_letter = self.create_alphabet()[random.randint(0, len(self.create_alphabet()) - 1)]
+        xpath = "//a[@class='pagination-item'][text()='" + self.random_letter + "']"
+        self.context.driver.find_element_by_xpath(xpath).click()
+
+    def compare_list(self):
+        letter_list = self.create_list_name()
+        if any(item.startswith(self.random_letter) for item in self.list_name) == letter_list:
```

Comment:
Can it be replaced with boolean result? Like 
"return any(item.startswith(self.random_letter) for item in self.list_name) == letter_list"
(I'm not sure the expression itself is correct though)

Reply:
done

Reply:
can we just
"return any(item.startswith(self.random_letter) for item in self.list_name) == letter_list"
?

Reply:
done